detekt-config.yml

```diff
@@ -204,7 +204,7 @@ naming:
   TopLevelPropertyNaming:
     active: true
     constantPattern: '[A-Z][_A-Za-z0-9]*'
-    propertyPattern: '[a-z][A-Za-z\d]*'
+    propertyPattern: '[A-Za-z][A-Za-z\d]*'
```

Comment:
@luongvo We need to update this pattern to resolve the warning on property declaration? 🤔

Reply:
@hoangnguyen92dn I think it's a good practice to update this, Here is a [ref](https://detekt.dev/docs/introduction/compose/)

Reply:
@Wadeewee @hoangnguyen92dn correct, here is the fix https://github.com/nimblehq/jetpack-compose-crypto/pull/66/commits/bc0cccb7814602f10e2b7b993829f98b5f264e1a